### ISSUE TITLE
Error during line item creative associaiton

### ISSUE THREAD

**Issue by dshore** [prebid/line-item-manager#25]
* line-item-manager version: all
* Python version: all
* Operating System: all

### Description

Creations of large numbers of line item creative associations (thousands) will result in unexpected errors during preparation, transmission and/or processing.

### What I Did

Ran config from @frjohan which has 399 line items and 18 creatives for 7,182 associations.

```
INFO:line_item_manager.gam_config:################################################################################
INFO:line_item_manager.gam_config:Bidder: name="Top Bid", code="hb"
INFO:line_item_manager.gam_config:Key: "hb_pb", Values: ['1.00', '1.50', '2.00', ..., '199.00', '199.50', '200.00']
INFO:line_item_manager.gam_config:############################################################
INFO:line_item_manager.gam_config:Media Type: "banner"
INFO:line_item_manager.gam_config:Line Items: CPMs(min=1.00, max=200.00, cnt=399)
INFO:line_item_manager.gam_config:Line Item Creative Associations: Creative Count=18
INFO:line_item_manager.gam_config:Auto-archiving Orders:
[2816257110]
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/googleads/common.py", line 991, in MakeSoapRequest
    return soap_service_method(
  File "/usr/local/lib/python3.8/site-packages/zeep/proxy.py", line 46, in __call__
    return self._proxy._binding.send(
  File "/usr/local/lib/python3.8/site-packages/zeep/wsdl/bindings/soap.py", line 135, in send
    return self.process_reply(client, operation_obj, response)
  File "/usr/local/lib/python3.8/site-packages/zeep/wsdl/bindings/soap.py", line 229, in process_reply
    return self.process_error(doc, operation)
  File "/usr/local/lib/python3.8/site-packages/zeep/wsdl/bindings/soap.py", line 317, in process_error
    raise Fault(
zeep.exceptions.Fault: Unknown fault occured

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/bin/line_item_manager", line 33, in <module>
    sys.exit(load_entry_point('line-item-manager', 'console_scripts', 'line_item_manager')())
  File "/home/app/line_item_manager/cli.py", line 160, in main
    cli() # pylint: disable=no-value-for-parameter
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.8/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/click/decorators.py", line 21, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/home/app/line_item_manager/cli.py", line 128, in create
    gam.create_line_items()
  File "/home/app/line_item_manager/gam_config.py", line 228, in create_line_items
    self._lica_objs.append(li_.create())
  File "/home/app/line_item_manager/gam_config.py", line 78, in create
    return LICA().create(recs, validate=True)
  File "/home/app/line_item_manager/gam_operations.py", line 41, in create
    getattr(self.svc(), self.create_method)(atts)
  File "/usr/local/lib/python3.8/site-packages/googleads/common.py", line 996, in MakeSoapRequest
    underlying_exception = e.detail.find(
TypeError: argument should be integer or bytes-like object, not 'str'
```

**Comment by dshore** [comment]
I have a fix in test that batches associations 100 at a time.  The above runs in ~15min. for me with the fix.

**Comment by dshore** [comment]
Fix is available as a dev release:

`$ pip uninstall line-item-manager
$ pip install https://github.com/prebid/line-item-manager/archive/master.zip
$ line_item_manager --version
line-item-manager version 0.1.14-dev1`

**Comment by dshore** [comment]
Released with: v0.1.15
